src/Exception/Database/DatabaseExceptionTrait.php

```diff
@@ -7,7 +7,7 @@
     /** @var string */
     protected $driver;
 
-    private static $driverNames = [
+    protected static $driverNames = [
```

Comment:
This should not be called directly from outside the trait, there is the `getPlatform()` for that.

Can you provide a trace for the exception you were seeing as I am unable to reproduce.